index.rst

```diff
@@ -11,14 +11,78 @@ Bharat-Sim's documentation!
    :caption: Contents:
 
 
-
-Indices and tables
-==================
-
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
 
-How does this lool?
+
+
+
+SIR Model with Agents
+=====================
+
+The Bharat-sim framework allows you to build agent-based models. Agents, are essentially the main component of the simulation. They are entities which are defined with a bunch of parameters and certain rules which they have to obey as the simulation runs.
```

Comment:
1. The first sentence doesn't catch the eye, and isn't saying anything special, it sounds a little trivial. Try to make it better. 
2. You don't need the comma after "Agents"
3. BharatSim isn't hyphenated, use the standard naming convention, please.
4. Try to avoid colloquialisms when you write: compare what you've written:

> They are entities which are defined with a bunch of parameters

with 

> They are entities that are defined with a _set_ of parameters

The second sounds a little more professional.

Reply:
Yeah, this is a just first draft. I'll keep on improving it.